src/components/Loading/Loading.less

```diff
@@ -9,17 +9,21 @@
   justify-content: center;
   align-items: center;
 
-  background-color: fade(@gray-dark, 75%);
+  // gray-dark #43484e in hex.
+  --faded-gray-dark: 67, 72, 78;
+  background-color: rgba(var(--faded-gray-dark), 0.75);
```

Comment:
In particular, the modal and loading background semi-opaque overlay should be unchanged.